packages/server/docker-compose.yml

```diff
@@ -1,14 +1,27 @@
 version: '3.4'
 
 volumes:
-  db-data:
+  pioneer-db-data:
+
+networks:
+  default:
+    name: joystream_default
+    external: true
 
 services:
-  db:
+  pioneer-api:
+    image: thesan/pioneer-backend # TODO change this to joystream/...
+    build:
+      context: ../..
+      dockerfile: backend.Dockerfile
+    ports: ['${PORT}:${PORT}']
+    depends_on: [pioneer-db]
+    env_file: [.env] # ensure `.env` exist
+    environment:
+      DATABASE_URL: postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@pioneer-db:5432/${POSTGRES_DB}
+
+  pioneer-db:
     image: postgres:12
-    ports:
-      - '127.0.0.1:${DB_PORT}:5432'
-    volumes:
-      - db-data:/var/lib/postgresql/data
-    env_file:
-      - .env
+    ports: ['127.0.0.1:${DB_PORT}:5432']
```

Comment:
Does the DB need to be exposed outside of docker-compose?

Reply:
I think it's better as `yarn workspace server docker:up` can be used to quickly work develop on the frontend (using a local backend). But to actually work on the backend this [Run locally](https://github.com/Joystream/pioneer/blob/feature/backend-deployments/packages/server/README.md#run-locally) guide is better (and still very simple) but it runs the api on a local network but the db with docker.

Which makes me realized I forgot to change the service name to `pioneer-db` on all these scripts !